command/flags/http.go

```diff
@@ -77,7 +77,7 @@ func (f *HTTPFlags) ServerFlags() *flag.FlagSet {
 func (f *HTTPFlags) NamespaceFlags() *flag.FlagSet {
 	fs := flag.NewFlagSet("", flag.ContinueOnError)
 	// TODO (namespaces) Do we want to allow setting via an env var? CONSUL_NAMESPACE
```

Comment:
This comment should be removed, since the env var is being added in this PR

Reply:
Removed.